<ui version="4.0" >
 <class>diskManipulator</class>
 <widget class="QDialog" name="diskManipulator" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Disk Manipulator</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="mediaComboBox" />
         </item>
         <item>
          <widget class="QToolButton" name="openImageButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset>openfile.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="newImageButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset>newfile.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="dirUpButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset>folderup.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="dirReloadButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset>folderreload.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="dirNewButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset>newfolder.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cwdLine" >
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="msxDirTable" >
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <property name="rowCount" >
          <number>2</number>
         </property>
         <property name="columnCount" >
          <number>3</number>
         </property>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="importButton" >
           <property name="text" >
            <string>&lt;&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="exportButton" >
           <property name="text" >
            <string>>></string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="hostDirUpButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset>folderup.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="hostDirReloadButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset>folderreload.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="hostDirNewButton" >
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset>newfolder.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="hostDirComboBox" >
             <property name="editable" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="hostDirTableWidget" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
